edx_proctoring/api.py

```diff
@@ -12,6 +12,8 @@
 import pytz
 import six
 
+from waffle import switch_is_active
```

Comment:
Aren't waffle switches deprecated?

Reply:
That's not my understanding of https://github.com/edx/open-edx-proposals/blob/master/oeps/oep-0017-bp-feature-toggles.rst#decision-map; I think our situation lines up pretty closely with #5.